test/unit/firebase.spec.ts

```diff
@@ -136,12 +136,21 @@ describe('Firebase', () => {
     });
 
     it('should initialize SDK given an application default credential', () => {
+      let credPath: string;
+      credPath = process.env.GOOGLE_APPLICATION_CREDENTIALS;
+      process.env.GOOGLE_APPLICATION_CREDENTIALS = path.resolve(__dirname, '../resources/mock.key.json');
       firebaseAdmin.initializeApp({
         credential: firebaseAdmin.credential.applicationDefault(),
       });
 
-      return firebaseAdmin.app().INTERNAL.getToken()
-        .should.eventually.have.keys(['accessToken', 'expirationTime']);
+      return firebaseAdmin.app().INTERNAL.getToken().then(token => {
+        if (credPath === 'undefined') {
```

Comment:
Is `credPath` actually equal to the string `undefined`?

I'm imagining you meant either:

```javascript
if (typeof credPath === 'undefined') { ... }
```

**or**

```javascript
if (credPath === undefined) { ... }
```